README.md

```diff
@@ -4,49 +4,15 @@ A virtual party space.
 
 ## Contributing
 
-### Installation
-```bash
-# Set up a virtualenv
-virtualenv venv
-source venv/bin/activate
-pip install -r requirements.txt
-
-# Make sure you have node/npm installed
-brew install node # on Linux: `apt install npm`
-
-# Make sure you have protoc installed
-brew install protobuf # on Linux: `apt install protobuf-compiler`
```

Comment:
Do users need to do anything to get these dependencies now (node and protoc)?

Reply:
This should stay. I'll restore